examples/preprocessing/plot_xdawn_denoising.py

```diff
@@ -34,7 +34,7 @@
 from mne import (io, compute_raw_covariance, read_events, pick_types,
                  Epochs)
 from mne.datasets import sample
-from mne.preprocessing import Xdawn
+from mne.preprocessing import XdawnTransformer
```

Comment:
wait I though we said we have Xdawn in preprocessing for denoising and that handles overlap. And have XdawnTransformer in decoding that is lighter and does not support overlap. So no need to pass events to XdawnTransformer in fit method.